packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -188,7 +190,9 @@ fi
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq
 %attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq/mnesia
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
-%dir %{_sysconfdir}/rabbitmq
+%attr(2755, -, -) %dir %{_sysconfdir}/rabbitmq
```

Comment:
I don't set  the `user` and `group` here : `-, -)` , but [reading better](http://ftp.rpm.org/max-rpm/s1-rpm-anywhere-specifying-file-attributes.html) it should be ok, it sets only the `g+s` that is exactly what we are looking for.